clone.js

```diff
@@ -0,0 +1,104 @@
+const readline = require('readline').createInterface({
```

Comment:
deze file is iets van de blueprint? moet hier niet zitten?

Reply:
Klopt, daar bestaat al een issue voor. Ik verwijder het.